src/components/FeatureViews/Onboarding/CourseUpload.js

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+import Box from "@material-ui/core/Box";
+import Container from "@material-ui/core/Container";
+import Typography from "@material-ui/core/Typography";
+import { makeStyles } from "@material-ui/core/styles";
+
+const useStyles = makeStyles((theme) => ({
+    Text: {
+        marginTop: '65px',
+        marginBottom: "30px;"
+    },
+    Subtitle: {
+        fontFamily: 'Arial, Helvetica Neue, Helvetica, sans-serif',
+        textAlign: "center",
+        marginTop: '40px',
+
+    },
+    uploadField: {
+        width: "250px",
+        height: "250px",
+        border: "2px dashed #28ABD5",
+        borderRadius: "10px",
+        margin: "auto"
+    },
+    uploadFieldText: {
+        marginTop: 110,
+        color: "#28ABD5"
+    }
+}));
+const CourseUpload = () => {
+    const classes = useStyles();
+
+    return (
+        <Container>
+        <Box className={classes.Text}>
+
+            <Typography 
```

Comment:
Same formatting for Typography as above, could put `<Typography variant="h3">` on its own line, same for line 40